### PR TITLE
chore(android): android build

### PR DESCRIPTION
**Pull request by WanderingHogan**
### What this PR does 📖
This build gets the basics back in to build the android version of Core, so we can start adding the capacitor modules we need to generate an APK we can submit to the app store.

### Which issue(s) this PR fixes 🔨
N/A

### Special notes for reviewers 🗒️
This should not affect the web version at all. In the future, updates to include capacitor modules will need to test to verify they dont break the web version, as well as fix the native versions.

### CODE REVIEW

File: android/app/proguard-rules.pro [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (stavares843) [inline android/app/proguard-rules.pro:1]: we don't need ProGuard

File: android/app/src/main/res/drawable/ic_launcher_background.xml [hunk 2]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (stavares843) [inline android/app/src/main/res/drawable/ic_launcher_background.xml:1]: we don't need these assets

File: android/app/src/test/java/com/getcapacitor/myapp/ExampleUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.getcapacitor.myapp;
```
Comment (stavares843) [inline android/app/src/test/java/com/getcapacitor/myapp/ExampleUnitTest.java:1]: we can ditch this test

File: android/app/proguard-rules.pro (hunk 1 above)
Comment (WanderingHogan) [reply android/app/proguard-rules.pro:1]: we probably will one day. https://developer.android.com/studio/build/shrink-code 
`Shrink, obfuscate, and optimize your app`

We will want to shrink and optimize, but not obfuscate. The source will be open anyways so it doesn't matter for that part.

File: android/app/src/main/res/drawable/ic_launcher_background.xml (hunk 2 above)
Comment (WanderingHogan) [reply android/app/src/main/res/drawable/ic_launcher_background.xml:1]: This is the default capacitor android build just to get the work started, and there is a ticket to fix the icons, which can be updated to fix the loading background.

**Pull request merged**